packages/legacy/tools/src/utils/server.ts

```diff
@@ -37,7 +37,12 @@ export async function startServer(): Promise<void> {
 
   return new Promise((resolve) => {
     // TODO:  move the logic for spawn the server to test-helper-server
-    const serverPath = '../../@webex/test-helper-server';
+    let serverPath;
```

Comment:
What does this change do?

Reply:
Basically we were setting a relative path for server that caused the original error. This was an attempt to fix that error which threw another error. 

That error requires more time to investigate and fix. Therefore, created a Jira here for the same: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-535983

I have reverted the changes here.